src/Common/ZooKeeper/ZooKeeperCommon.cpp

```diff
@@ -175,6 +207,19 @@ void ZooKeeperCreateRequest::writeImpl(WriteBuffer & out) const
     Coordination::write(flags, out);
 }
 
+size_t ZooKeeperCreateRequest::sizeImpl() const
+{
+    /// See https://github.com/ClickHouse/clickhouse-private/issues/3029
+    if (path.starts_with("/clickhouse/tables/") && path.find("/parts/") != std::string::npos)
```

Comment:
It's a logging from private repo https://github.com/ClickHouse/clickhouse-private/pull/3969
do we need it here? and it's weird to log in `sizeImpl`

Reply:
copy paste man... good catch